x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/jwt/JwtTestCase.java

```diff
@@ -238,6 +239,55 @@ protected RealmConfig buildRealmConfig(
         };
     }
 
+    public static String printRealmSettings(final RealmConfig config, final boolean includeSecureSettings) {
+        final StringBuilder sb = new StringBuilder("\n===\nelasticsearch.yml settings\n===\n");
+        int numRegularSettings = 0;
+        for (final Setting.AffixSetting<?> setting : JwtRealmSettings.getNonSecureSettings()) {
+            final String key = RealmSettings.getFullSettingKey(config, setting);
+            if (key.startsWith("xpack.") && config.hasSetting(setting)) {
+                sb.append(key).append(": ").append(config.getSetting(setting)).append('\n');
+                numRegularSettings++;
+            }
+        }
+        if (numRegularSettings == 0) {
+            sb.append("Not found.\n");
+        }
```

Comment:
Same for this method.

Reply:
Moved to JwtRealmGenerateTests. Resolved.